windows/src/support/convert-iso6393-to-bcp47-for-keyboards/cvt.dpr

```diff
@@ -7,10 +7,10 @@ program cvt;
 uses
   System.SysUtils,
   cv in 'cv.pas',
-  Keyman.System.Standards.ISO6393ToBCP47Registry in 'C:\Projects\keyman\open\windows\src\global\delphi\standards\Keyman.System.Standards.ISO6393ToBCP47Registry.pas',
+  Keyman.System.Standards.ISO6393ToBCP47Registry in 'C:\Projects\keyman\open\common\windows\delphi\standards\Keyman.System.Standards.ISO6393ToBCP47Registry.pas',
```

Comment:
Using relative paths would be nicer :smile:

Reply:
Indeed. However as this build was clearly broken before I will leave this to either:

1. if/when we ever need this project again we can fix up then, or
2. a cleanup for the support folder to remove obsolete projects...